### PR TITLE
Added Turkish Language Support

### LINKED ISSUE #1193: Turkish language support
This issue will be used to add Turkish language support to Teslamate

### PR DESCRIPTION
**Pull request by neocorp**
Added support for Turkish in Teslamate as a display language, fixes #1193

Hope everything's correct.

### CODE REVIEW

File: priv/gettext/tr/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+#: lib/teslamate_web/live/geofence_live/form.ex:126
+#: lib/teslamate_web/live/geofence_live/form.html.leex:4 lib/teslamate_web/live/geofence_live/index.ex:22
+#: lib/teslamate_web/live/geofence_live/index.html.leex:4 lib/teslamate_web/templates/layout/root.html.leex:56
+msgid "Geo-Fences"
+msgstr "Coğrafi-Sınırlar"
+
+#, elixir-format
+#: lib/teslamate_web/live/settings_live/index.html.leex:49
+msgid "Uykuya Dalmadan Önce Boşta Geçen Zaman"
+msgstr ""
```
Comment (adriankumpf) [inline priv/gettext/tr/LC_MESSAGES/default.po:148]: I think something went wrong here by accident
I think you accidentally overwrote the "msgid" here.

```suggestion
msgid "Idle Time Before Trying to Sleep"
msgstr "Uykuya Dalmadan Önce Boşta Geçen Zaman"
```

File: priv/gettext/tr/LC_MESSAGES/default.po [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+msgstr "ideal"
+
+#, elixir-format
+#: lib/teslamate_web/live/settings_live/index.html.leex:148
+msgid "rated"
+msgstr "hesaplanan"
+
+#, elixir-format
+#: lib/teslamate_web/live/car_live/summary.html.leex:123
+msgid "Menzil (ideal)"
```
Comment (adriankumpf) [inline priv/gettext/tr/LC_MESSAGES/default.po:284]: Same here

```suggestion
msgid "Range (ideal)"
```

File: priv/gettext/tr/LC_MESSAGES/default.po (hunk 1 above)
Comment (neocorp) [reply priv/gettext/tr/LC_MESSAGES/default.po]: fixed in new commit

File: priv/gettext/tr/LC_MESSAGES/default.po (hunk 2 above)
Comment (neocorp) [reply priv/gettext/tr/LC_MESSAGES/default.po]: fixed in new commit

**Review by adriankumpf — APPROVED** [review]
Awesome. Thank you!

**Pull request merged**